super_csv/serializers.py

```diff
@@ -38,5 +44,22 @@ def get_related_queryset(cls, queryset):
         return queryset.select_related('user')
 
     def get_data(self, operation):
-        data = json.load(operation.data)
-        return CSVOperationDataSerializer(data).data
+        """
+        Get data
+        """
+        try:
+            data = json.load(operation.data)
+        except (FileNotFoundError, ValueError) as error:
```

Comment:
I am not expecting to see `FileNotFoundError` but leaving it for observation.

Reply:
I am not raising exception because `data` is a `SerializerMethodField` and invoke with `many=true`. I am not sure if I can raise exception without interrupt the data that is valid.

Reply:
Also interesting find. Most errors; if not all, should be expired file. File expiration decision was to just removed the file and kept the records. Here is the [expire file code](https://github.com/edx/super-csv/blob/ea76ab38abca0c0237a869c98bde012773e185ed/super_csv/models.py#L79) and [test expire file code](https://github.com/edx/super-csv/blob/ea76ab38abca0c0237a869c98bde012773e185ed/tests/test_models.py#L27).